src/components/users/LoginForm/LoginForm.js

```diff
@@ -1,24 +1,54 @@
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+
 import "style/common/_common.scss";
 import CommonInput from "components/common/CommonInput";
+import { userActions } from "actions/userActions";
+import { userActionTypesConstants } from "constants/userConstants";
 
 import "./login-form.scss";
 
-const LoginForm = () => (
-  <form className="login-form">
-    <CommonInput
-      label="Email"
-      inputType="email"
-      placeHolder="Type your email"
-    />
-    <CommonInput
-      label="Password"
-      inputType="password"
-      placeHolder="Type your password"
-    />
-    <button type="button" className="regular-text">
-      Log in
-    </button>
-  </form>
-);
+const { USER_REQUEST } = userActionTypesConstants;
+
+const LoginForm = () => {
+  const dispatch = useDispatch();
+
+  const [inputs, setInputs] = useState({ email: "", password: "" });
+  const { email, password } = inputs;
+
+  const handleChange = ({ target: { name, value } }) => {
+    setInputs((inputs) => ({ ...inputs, [name]: value }));
+  };
+
+  const handleSubmit = (e) => {
+    dispatch({ type: USER_REQUEST });
```

Comment:
This dispatch should be included inside your login action. I recommend you to dispatch only the next one action and before making the axios request do this dispatch